source/utilities.cc

```diff
@@ -430,6 +430,47 @@ namespace aspect
 
 
 
+    template <int dim>
+    std::vector<Point<dim>>
+                         get_unit_support_points_for_velocity(const SimulatorAccess<dim> &simulator_access)
+    {
+      std::vector<Point<dim>> unit_support_points;
+      if ( !simulator_access.get_parameters().use_locally_conservative_discretization )
+        {
+          return simulator_access.get_fe().get_unit_support_points();
+        }
+      else
+        {
+          //special case for discontinuous pressure elements, which lack unit support points
+          std::vector<Point<dim>> unit_support_points;
+          const unsigned int dofs_per_cell = simulator_access.get_fe().dofs_per_cell;
+          for (unsigned int dof=0; dof < dofs_per_cell; ++dof)
+            {
+              // base will hold element, base_index holds node/shape function within that element
+              const unsigned int
+              base       = simulator_access.get_fe().system_to_base_index(dof).first.first,
+              base_index = simulator_access.get_fe().system_to_base_index(dof).second;
```

Comment:
I know this is just moved code, but now I cant unsee it :-).
```suggestion
              const unsigned int base_element = simulator_access.get_fe().system_to_base_index(dof).first.first,
              const unsigned int base_index = simulator_access.get_fe().system_to_base_index(dof).second;
```